llama-index-core/llama_index/core/indices/document_summary/base.py

```diff
@@ -201,12 +201,20 @@ def _add_nodes_to_index(
             )
             summary_response = cast(Response, summary_response)
             metadata = doc_id_to_nodes.get(doc_id, [TextNode()])[0].metadata
```

Comment:
inside of calling `get(doc_id, ...)` so many times, it would be much cleaner to only call this once and use it where its needed

Reply:
done.